src/test/java/org/apache/maven/plugins/checkstyle/exec/CheckstyleExecutorTest.java

```diff
@@ -19,10 +19,13 @@
 package org.apache.maven.plugins.checkstyle.exec;
 
 import org.apache.maven.plugin.testing.AbstractMojoTestCase;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
 
 public class CheckstyleExecutorTest extends AbstractMojoTestCase {
+    @Test
```

Comment:
Still extends `AbstractMojoTestCase` so it is JUnit3 test, so we can not use annotations from JUnit5